--- urbanonto_dump_202102181134.sql.orig	2021-02-19 09:04:51.382671070 +0100
+++ urbanonto_dump_202102181134.sql	2021-02-19 09:34:54.231251063 +0100
@@ -10,7 +10,7 @@
 SET idle_in_transaction_session_timeout = 0;
 SET client_encoding = 'UTF8';
 SET standard_conforming_strings = on;
-SELECT pg_catalog.set_config('search_path', '', false);
+SELECT pg_catalog.set_config('search_path', 'public', false);
 SET check_function_bodies = false;
 SET xmloption = content;
 SET client_min_messages = warning;
@@ -35,10 +35,10 @@
 ALTER SCHEMA ontology_sources OWNER TO postgres;
 
 --
--- Name: postgis; Type: EXTENSION; Schema: -; Owner: -
+-- Name: postgis; Type: EXTENSION; Schema: public; Owner: -
 --
 
-CREATE EXTENSION IF NOT EXISTS postgis WITH SCHEMA ontology;
+CREATE EXTENSION IF NOT EXISTS postgis WITH SCHEMA public;
 
 
 --
@@ -147,7 +147,7 @@
 
 CREATE TABLE ontology.locations (
     identifiers integer NOT NULL,
-    the_geom ontology.geometry NOT NULL,
+    the_geom geometry NOT NULL,
     location_type_identifiers integer NOT NULL,
     names text
 );
@@ -422,7 +422,7 @@
 
 CREATE TABLE ontology_sources.locations (
     identifiers integer NOT NULL,
-    the_geom ontology.geometry NOT NULL
+    the_geom geometry NOT NULL
 );
 
 
